OloEditor/src/EditorLayer.cpp

```
@@ -131,6 +135,19 @@ namespace OloEngine
             m_Framebuffer->ClearAttachment(1, -1);
         }
 
+        // Upload brush preview UBO for terrain shader
+        {
+            ShaderBindingLayout::BrushPreviewUBO brushData{};
```

Clang (14.0.6)

[warning] variable 'brushData' is not initialized

(cppcoreguidelines-init-variables)